packages/api/cms-api/src/page-tree/dto/page-tree-node.input.ts

```diff
@@ -66,6 +66,11 @@ export abstract class PageTreeNodeBaseUpdateInput {
     @IsOptional()
     @IsBoolean()
     hideInMenu?: boolean;
+
+    @Field({ nullable: true, defaultValue: false })
+    @IsOptional()
+    @IsBoolean()
+    createRedirectFromOldToNewSlug: boolean;
```

Comment:
I'd call this `createAutomaticRedirects` or `createAutomaticRedirectsOnSlugChange`, because it may create multiple redirects (one for every child node and the parent node)